components/utils.py

```diff
@@ -158,6 +158,10 @@ def move(source, destination):
     remove(source)
 
 
```

Comment:
This function actually allows me to copy the dir without having to create it first. Does this make sense?